src/pathfinder/yapf/yapf_costrail.hpp

```diff
@@ -31,48 +35,36 @@ class CYapfCostRailT : public CYapfCostBase {
 		TileType    tile_type;
 		RailType    rail_type;
 
-		TILE()
-		{
-			tile = INVALID_TILE;
-			td = INVALID_TRACKDIR;
-			tile_type = MP_VOID;
-			rail_type = INVALID_RAILTYPE;
-		}
+		TILE() : tile(INVALID_TILE), td(INVALID_TRACKDIR), tile_type(MP_VOID), rail_type(INVALID_RAILTYPE) { }
 
-		TILE(TileIndex tile, Trackdir td)
-		{
-			this->tile = tile;
-			this->td = td;
-			this->tile_type = GetTileType(tile);
-			this->rail_type = GetTileRailType(tile);
-		}
+		TILE(TileIndex tile, Trackdir td) : tile(tile), td(td), tile_type(GetTileType(tile)), rail_type(GetTileRailType(tile)) { }
 	};
 
 protected:
 	/**
 	 * @note maximum cost doesn't work with caching enabled
 	 * @todo fix maximum cost failing with caching (e.g. FS#2900)
 	 */
-	int m_max_cost;
-	bool m_disable_cache;
-	std::vector<int> m_sig_look_ahead_costs;
+	int max_cost;
+	bool disable_cache;
+	std::vector<int> sig_look_ahead_costs;
 
 public:
-	bool          m_stopped_on_first_two_way_signal;
-protected:
+	bool stopped_on_first_two_way_signal;
 
+protected:
 	static const int s_max_segment_cost = 10000;
```

Comment:
```suggestion
	static constexpr int MAX_SEGMENT_COST = 10000;
```
Not sure if static is needed